posts/views.py

```diff
@@ -39,6 +39,11 @@ def list(self, request) -> Response:
 class LikeDislikeViewSet(viewsets.ModelViewSet):
     permission_classes = (IsAuthenticated,)
 
+    def get(self, request, pk: int):
+        is_like = "dislike" not in request.path
+        count = LikeDislike.objects.filter(post_id=pk, is_like=is_like).count()
+        return Response({"count": count}, status=status.HTTP_200_OK)
```

Comment:
We can rather have a view that returns both numbers of likes and dislikes. It will help to decrease client app calls count

Reply:
Done